### ISSUE TITLE
Slight incompatibility with DuckDB

### ISSUE THREAD

**Issue by max-sixty** [PRQL/prql#658]
Currently, querying files in DuckDB doesn't quite work without an alias, because:

```elm
from products.csv
filter supplierID == 1
```

compiles to:

```sql
SELECT
  products.csv.*
FROM
  products.csv
WHERE
  supplierID = 1
```

And actually `products.csv.*` isn't valid.

We could get around this with just `select *` when that's all that's needed. This would be valid in every dialect, I think.

**Comment by aljazerzen** [comment]
With the new changes this errors out:

```
from products.csv
filter supplierID == 1
Error: 
   ╭─[-:1:6]
   │
 1 │ from products.csv
   ·      ──────┬─────  
   ·            ╰─────── Unknown name products.csv
───╯
```
We should include a hint here, saying "if your table name includes dots or has schema name, you have to enclose it in backticks:

```elm
from `products.csv`
filter supplierID == 1
```
```sql
SELECT
  *
FROM
  products.csv
WHERE
  "supplierID" = 1
```
```

**Comment by aljazerzen** [comment]
Closed with #593.

```elm
from `products.csv`
filter supplierID == 1
```
```sql
SELECT
  *
FROM
  products.csv
WHERE
  "supplierID" = 1
```
